Bump teslajsonpy to 2.4.1

shred86 opened the pull request:
Adds grid and home (load) power sensors for energy sites (solar systems).

alandtse reviewed (changes requested):
Thanks. Can you also make sure to update pyproject.toml and do a poetry update instead of modifying poetry.lock directly?

shred86 commented:
Sorry about that - clearly have no idea what I'm doing with poetry. 😆 

So I just realized the only reason the solar sensor is being recreated is because I renamed it from `solar panel` to `solar power` to align the naming convention, `solar power`, `grid power`, `load power`. Instead of creating a breaking change, I can simply just retain the `solar panel` name so the `unique_id` in Home Assistant doesn't change...

shred86 commented:
Just made a [pull request](https://github.com/zabuldon/teslajsonpy/pull/336) over on teslajsonpy to revert the name. Sorry for the hassle and I should have caught this earlier but this will prevent the breaking change.

alandtse commented:
Thanks for the patience. Just returned from vacation. 2.4.1 is pushed with the reversion.

shred86 commented:
Welcome back! Just updated the PR to include version 2.4.1 for teslajsonpy. I also updated the version of this integration to 2.4.0 in the manifest.json which is what I'm assuming it will be.

alandtse commented:
You need to run `poetry update`. It should auto update dependencies and find 2.4.1.

alandtse commented on pyproject.toml:
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tesla"
-version = "2.3.1"
```
Don't bump the component version. That's done by CI.

shred86 replied:
Updated! Sorry about that.

alandtse commented on custom_components/tesla_custom/manifest.json:
```diff
@@ -22,5 +22,5 @@
     }
   ],
   "iot_class": "cloud_polling",
-  "version": "2.3.1"
```
Don't change this.

shred86 replied:
Updated! Sorry about that.

shred86 commented:
Sorry it wasn't updating to 2.4.1 when doing `poetry update`. It finally did after it was published for 13 minutes... guess I was just being impatient. 😄
